### ISSUE TITLE
Make filter sidebar opening and closing more obvious

### ISSUE THREAD

**Issue by hafen** [trelliscope/trelliscopejs-lib#1019]
It may not be entirely clear that the "x" icon in the filters button closes the sidebar.

<img width="285" alt="is x clear enough to close sidebar" src="https://github.com/trelliscope/trelliscopejs-lib/assets/1275592/c4e0bb7f-dc47-4ee7-961d-24df54321ec6">

I wonder if something more like this would be more obvious:

<img width="431" alt="image" src="https://github.com/trelliscope/trelliscopejs-lib/assets/1275592/0248c8ae-de27-4a99-97f3-2735f8c94d29">

Basically a circular button with a left chevron that overlaps the sidebar boundary.
